### PR TITLE
Samoa

### PR DESCRIPTION
**Pull request by holzman**
Add a FIPS for American Samoa (this is crashing the app now).

### CODE REVIEW

File: src/CovidCountyDash.jl [hunk 1]
```diff
@@ -89,6 +89,8 @@ function download_and_preprocess()
                     29998
                 elseif state == "Missouri" && county == "Joplin"
                     29997
+		elseif state == "American Samoa"
+		    03999
```
Comment (mbauman) [inline src/CovidCountyDash.jl]: I think this should be 60999?  Do we also need its population?

Comment (holzman) [reply src/CovidCountyDash.jl]: Oh, 60999 should be right - I looked up the wrong row in the FIPS table.  And yes, we should fill in the population; I can update this PR for that when I have time.

Comment (mbauman) [reply src/CovidCountyDash.jl]: Aha, yeah, and it looks like we don't need to special-case its FIPS here — once it has a population entry the unknown FIPS will get assigned appropriately by the next branch here.

Comment (holzman) [reply src/CovidCountyDash.jl]: Yeah, my fault for not looking a little deeper.  Please recheck the samoa branch, this should be right now.

**Comment by mbauman** [comment]
Thanks!

**Comment by mbauman** [comment]
... and it's back up! I had noticed the site was down but hadn't gotten a chance to dig into why — this was greatly helpful in both identifying the problem and helping solve it.

**Pull request merged**